### PR TITLE
bumped version to 1.0

### PR DESCRIPTION
**Pull request by oetterer**
also: cleaned-up my leftovers in subobject-001.json

### CODE REVIEW

File: SemanticScribunto.php [hunk 1]
```diff
@@ -50,7 +50,7 @@ public static function load() {
 	 */
 	public static function initExtension() {
 
-		define( 'SMW_SCRIBUNTO_VERSION', '1.0.0-alpha' );
+		define( 'SMW_SCRIBUNTO_VERSION', '1.0' );
```
Comment (mwjames) [inline SemanticScribunto.php]: For versioning we follow the semver, which means the full 1.0.0 number. Further more doing a tidy and a release prep should be two distinct tasks.

Comment (mwjames) [inline SemanticScribunto.php]: http://semver.org/

**Comment by oetterer** [comment]
ok. got it. sry

just the tidy, then... :)

**Comment by mwjames** [comment]
> ok. got it. sry. Got a little overexcited...

Don't worry about it, this why we do reviews.

**Pull request merged**